### PR TITLE
Awesome books: with ES6

### PR DESCRIPTION
**Pull request by berylcl**
In this project i was able to :
:heavy_check_mark:Use ES6 syntax.
:heavy_check_mark:Divide my code into modules.
:heavy_check_mark:Create a separate file for each module in the modules directory.
:heavy_check_mark:Import modules into the index.js file. 
:heavy_check_mark:Keep all funcionalities of the app without errors.
:heavy_check_mark:Refactor my functions to arrow functions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,133 @@
...
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Awesome-books-with-ES6] <a name="about-project"></a>
+
+> **[Awesome Books]** is a...This is a simple html page which is contains a dynamic form about Awesome books, books title, and the author. I have used HTML and Javascript for this project and you can use it in your local machine just by cloning this repo.
+
```
Comment (wayungi) [inline README.md]: - [ OTPIONAL] Kindly edit this document to remove all the `>` marks. These mark out the content as a guide. 
guide

File: style.css
Comment (wayungi) [inline style.css:1]: - [ ] Please rename this file to **index.css** or add an `index.css` file as clearly stated by the requirements
file

File: modules/books.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+export default class Bookdata {
+  constructor(id, title, author) {
+    this.Id = id;
+    this.Title = title;
+    this.Author = author;
```
Comment (wayungi) [inline modules/books.js]: - [ ] Javascript variables are named following **[camelCase](https://www.syncfusion.com/blogs/post/10-javascript-naming-conventions-every-developer-should-know.aspx)** convention. With this convention, `this.Id` would be `this.id`. Please stick to this convention when naming functions & variables. code written following convention is clear & easy to read. 😄

**Review by wayungi — CHANGES_REQUESTED** [review]
Hi @berylcl , 

image


**Congratulations on the good job done so far!** 🤝 

There are some issues that you still need to work on to go to the next project but you are almost there! 😄 

## Highlights ⚡ ⚡ 
- [ ] All linter checks passing ✔️ 
- [ ]  Correct GitHub flow ✔️ 
- [ ] Descriptive PR body ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AlphaNtihinduka — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 
## Hi @berylcl 🙋‍♂️, 

Your project is complete!  There is nothing else to say other than... Great job 👏 

approved-accepted

Congratulations! 🎉 
TO HIGHLIGHT:

- Your project is professional :heavy_check_mark: 
- Your repo is professional :heavy_check_mark: 
- Well-structured files :heavy_check_mark: 
- Professional README.md file 💯
- No linter errors :heavy_check_mark: 

Now you can merge this branch and move on :1st_place_medal: 


## Optional suggestions
- Kindly refer to the inline optional suggestions made. Ensure to apply them to have a professional project that suits recruiters' needs.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK WITH THE COMING ONES :crossed_fingers: :four_leaf_clover:

**Pull request merged**